cli/src/pcluster/resources/imagebuilder/parallelcluster_validate.yaml

```diff
@@ -0,0 +1,454 @@
+                echo "true"
+              else
+                echo "false"
+              fi
+
+      - name: LustreSupported
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              ARCHITECTURE='{{ validate.OperatingSystemArchitecture.outputs.stdout }}'
+              OS='{{ validate.OperatingSystemName.outputs.stdout }}'
+              if [ ${!ARCHITECTURE} == 'arm64' ] && [[ ${!OS} =~ ^(ubuntu(18|20)04|centos8|alinux2)$ ]] || [ ${!ARCHITECTURE} == 'x86_64' ]; then
+                echo "true"
+              else
+                echo "false"
+              fi
+
+      ### version ###
+      - name: PythonVersion
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "3.7.10"
+
+      - name: NvidiaDriverVersion
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "450.80.02"
+
+      - name: CudaVersion
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "11.0"
+
+      - name: ArmPLVersion
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "21.0.0"
+
+      - name: ArmPLGCCVersion
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "9.3"
+
+      ### utils ###
+
+      - name: NvSwitches
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              nvswitches=$(lspci -d 10de:1af1 | wc -l)
+              echo "$nvswitches"
+
+      ### path ###
+      - name: SystemPyenvRoot
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "/opt/parallelcluster/pyenv"
+
+      - name: CookbookVirtualenvPath
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "{{ validate.SystemPyenvRoot.outputs.stdout }}/versions/{{ validate.PythonVersion.outputs.stdout }}/envs/cookbook_virtualenv"
+
+      - name: NodeVirtualenvPath
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo "{{ validate.SystemPyenvRoot.outputs.stdout }}/versions/{{ validate.PythonVersion.outputs.stdout }}/envs/node_virtualenv"
+
+      ### validate && test ###
+      # AWS CLI
+      - name: AWSCli
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -evx
+              export PATH="/usr/local/bin:/usr/bin/:${!PATH}"
+
+              echo "execute awscli as user"
+              aws --version
+
+              echo "execute awscli as root"
+              {{ validate.CookbookVirtualenvPath.outputs.stdout }}/bin/aws --version
```

Comment:
ok it's executed as root, but it's using the `CookbookVirtualenvPath`. Probably we should split the test